frontend/src/views/camp/CampProgram.vue

```diff
@@ -19,33 +19,29 @@ Show all activity schedule entries of a single period.
         <v-icon v-if="listFormat">mdi-calendar-month</v-icon>
         <v-icon v-else>mdi-menu</v-icon>
       </v-btn>
-      <template v-if="!(period() && !period()._meta.loading)">
-        <v-skeleton-loader v-if="listFormat" type="list-item-avatar-two-line@2" class="py-2" />
-        <v-skeleton-loader v-else type="table" />
-      </template>
-      <template v-if="firstPeriodLoaded">
-        <picasso
-          v-show="!listFormat"
-          class="mx-2 ma-sm-0 pa-sm-2"
-          :period="period"
-          :start="Date.parse(period().start)"
-          :end="Date.parse(period().end)" />
-        <activity-list
-          v-show="listFormat"
-          :period="period" />
-      </template>
-      <v-btn
-        :fixed="$vuetify.breakpoint.xs"
-        :absolute="!$vuetify.breakpoint.xs"
-        dark
-        fab
-        style="z-index: 3"
-        bottom
-        right
-        class="fab--bottom_nav float-right"
-        color="red">
-        <v-icon>mdi-plus</v-icon>
-      </v-btn>
+      <schedule-entries :period="period" :show-button="true">
+        <template v-slot:default="slotProps">
+          <template v-if="slotProps.loading">
+            <v-skeleton-loader v-if="listFormat" type="list-item-avatar-two-line@2" class="py-2" />
+            <v-skeleton-loader v-else type="table" />
+          </template>
+          <template v-else>
+            <picasso
+              v-show="!listFormat"
```

Comment:
I know this was in before, but @manuelmeister why don't we use `v-if` / `v-else` here?

Reply:
I think this was to reduce loading time of the page. You most certainly wan't to see the picasso and if you change the viewmode, then the whole DOM needs to be build again, instead of just toggling `display`.

Reply:
Hm okay. Maybe we'll have another look at this in #666